megalinter/descriptors/salesforce.megalinter-descriptor.yml

```diff
@@ -151,37 +150,40 @@ linters:
     cli_config_arg_name: ".eslintrc-aura.json"
     cli_executable: sf
     cli_lint_extra_args:
-      - "scanner:run"
-      - "--engine"
+      - "code-analyzer"
+      - "run"
+      - "--rule-selector"
       - "eslint"
+      - "--workspace"
+      - "."
       - "--target"
       - "**/aura/**/*.js"
       - "--severity-threshold"
       - "2"
-      - "--outfile"
-      - "{{REPORT_FOLDER}}/sfdx-scanner-report-aura.csv"
-      - "--format"
-      - "csv"
+      - "--output-file"
+      - "{{REPORT_FOLDER}}/code-analyzer-report-aura.csv"
+      - "--view"
+      - "table"
     cli_lint_mode: project
     cli_help_extra_args:
-      - "scanner:run"
+      - "code-analyzer run"
```

Comment:
Maybe (I'm not sure though) that the two args can be together. I may depend by platform, but the container only runs on Linux, so only one OS to make it work